service/worker/scanner/scanner.go

```diff
@@ -123,25 +118,42 @@ func New(
 
 // Start starts the scanner
 func (s *Scanner) Start() error {
+	backgroundActivityContext := context.WithValue(context.Background(), scannerContextKey, s.context)
+	backgroundActivityContext = context.WithValue(backgroundActivityContext, executions.ScannerContextKey, executions.ScannerContext{
+		Resource:                     s.context.Resource,
+		Scope:                        s.context.Resource.GetMetricsClient().Scope(metrics.ExecutionsScannerScope),
+		ScannerWorkflowDynamicConfig: s.context.cfg.ExecutionScannerConfig,
+	})
+	backgroundActivityContext = context.WithValue(backgroundActivityContext, executions.FixerContextKey, executions.FixerContext{
+		Resource: s.context.Resource,
+		Scope:    s.context.Resource.GetMetricsClient().Scope(metrics.ExecutionsFixerScope),
+	})
```

Comment:
nit: only set the scanner context when scanner is enabled? `ExecutionsScannerEnabled` flag requires restart anyway I think?

Reply:
Will address in next diff.